templates/chat/roll-result.html

```diff
@@ -1,272 +1,185 @@
-	data-item-id="{{item.id}}"
-	data-dice-mod="{{diceModifier}}"
-	data-dice-modifier="{{diceModifier}}"
-	data-flat-dice-modifier="{{flatDiceModifier}}"
-	data-flat-threshold-modifier="{{flatThresholdModifier}}"
-	data-ten-only-one-die="{{dices.tenOnlyOneDie}}"
-	data-raw-value="{{rawValue}}"
-	data-result="{{dice.total}}"
-	data-is-success="{{isSuccess}}"
-	data-is-failure="{{isFailure}}"
-	data-is-unknown="{{isUnknown}}"
-	data-luck-spent="{{luckSpent}}"
-	data-total-luck-spent="{{totalLuckSpent}}"
-	data-gm-difficulty-regular="{{gmDifficultyRegular}}"
-	data-gm-difficulty-hard="{{gmDifficultyHard}}"
-	data-gm-difficulty-extreme="{{gmDifficultyExtreme}}"
-	data-gm-difficulty-critical="{{gmDifficultyCritical}}"
-	data-hide-dice-result="{{hideDiceResult}}"
-	data-deny-luck="{{denyLuck}}"
-	data-deny-push="{{denyPush}}"
-	{{#if tokenId}}data-token-id="{{tokenId}}"{{/if}}>
+<div class="coc7 chat-card chat-card-v2 item-card roll-card roll-result" data-actor-name="{{actorName}}" data-standby="{{standby}}" data-standby-text="{{standbyText}}" data-standby-text="{{standbyRightIcon}}" data-parent-uuid="{{parentUuid}}" data-uuid="{{uuid}}" data-context="{{context}}" data-raw-value="{{rawValue}}" data-characteristic="{{characteristic}}" data-attribute="{{attribute}}" data-skill-id="{{skill.id}}" data-success-required="{{successRequired}}" data-flagged-for-development="{{flaggedForDevelopment}}" data-forced="{{forced}}" data-forced-success="{{forcedSuccess}}" data-forced-failure="{{forcedFailure}}" data-result-type="{{resultType}}" data-details="{{details}}" data-can-be-pushed="{{canBePushed}}" data-pushing="{{pushing}}" data-has-enough-luck="{{hasEnoughLuck}}" data-luck-needed="{{luckNeeded}}" data-can-increase-success="{{canIncreaseSuccess}}" data-has-malfunction="{{hasMalfunction}}" data-malfunction-txt="{{malfunctionTxt}}" data-token-id="{{tokenId}}" data-roll-type="{{rollType}}" data-actor-key="{{actorKey}}" data-is-blind="{{isBlind}}" data-side="{{side}}" data-action="{{action}}" data-ref-message-id="{{referenceMessageId}}" data-success-level="{{successLevel}}" data-difficulty="{{difficulty}}" data-actor-id="{{actor.id}}" data-item-id="{{item.id}}" data-dice-mod="{{diceModifier}}" data-dice-modifier="{{diceModifier}}" data-flat-dice-modifier="{{flatDiceModifier}}" data-flat-threshold-modifier="{{flatThresholdModifier}}" data-ten-only-one-die="{{dices.tenOnlyOneDie}}" data-raw-value="{{rawValue}}" data-result="{{dice.total}}" data-is-success="{{isSuccess}}" data-is-failure="{{isFailure}}" data-is-unknown="{{isUnknown}}" data-luck-spent="{{luckSpent}}" data-total-luck-spent="{{totalLuckSpent}}" data-gm-difficulty-regular="{{gmDifficultyRegular}}" data-gm-difficulty-hard="{{gmDifficultyHard}}" data-gm-difficulty-extreme="{{gmDifficultyExtreme}}" data-gm-difficulty-critical="{{gmDifficultyCritical}}" data-hide-dice-result="{{hideDiceResult}}" data-deny-luck="{{denyLuck}}" data-deny-push="{{denyPush}}" {{#if tokenId}}data-token-id="{{tokenId}}" {{/if}}>
```

Comment:
I imagine is the auto formatter, but this was much more readable on the previous version with one data per line